autorest/code_generator.py

```diff
@@ -66,13 +65,15 @@ def _build_exceptions_set(self, yaml_data):
                 for exception in operation['exceptions']:
                     if not exception.get('schema'):
                         continue
-                    exceptions_set.add(exception['schema']['language']['default']['name'])
+                    exceptions_set.add(exception['schema']['language']['python']['name'])
         return exceptions_set
 
     def _create_code_model(self, yaml_code_model):
+        NameConverter.convert_yaml_names(yaml_code_model)
```

Comment:
Conceptually, this one should be called in "process" just after we read the file from autorest_api. I know that's picky, but it's not part of the code model (hence should not be part of "create_code_model").

Reply:
yeah that makes a lot of sense, moving it thanks!